fusedrug/data/protein/structure/split_scfv_chain.py

```diff
@@ -0,0 +1,180 @@
+from jsonargparse import CLI
+from fusedrug.data.protein.structure.structure_io import (
+    load_pdb_chain_features,
+    save_structure_file,
+)
+from typing import Optional, Sequence
+from os.path import isfile, join, dirname, basename
+import os
+import sys
+import subprocess
+import threading
+
+
+def main(
+    *,
+    input_pdb_path: str,
+    input_scfv_chain_id: str,
+    output_pdb_path_extensionless: str,
+    output_heavy_chain_id: Optional[str] = "H",
+    output_light_chain_id: Optional[str] = "L",
+    passthrough_chains: Optional[str] = None,
+    cleanup_temp_files: bool = True,
+) -> None:
+    """
+
+    Takes an input PDB files and splits it into separate files, one per describe chain, allowing to rename the chains if desired
+
+    Args:
+    input_pdb_path:
+
+    passthrough_chains: optional, will be "pass through", '_' separated if you want multiple
+
+    """
+
+    if passthrough_chains is not None:
+        passthrough_chains = passthrough_chains.split("_")
+
+    loaded_scfv = load_pdb_chain_features(input_pdb_path, input_scfv_chain_id)
+
+    scfv_seq = loaded_scfv["aasequence_str"]
+
+    safety = f"_{os.getpid()}_{threading.get_ident()}"
+
+    scfv_sequence_filename = join(
+        dirname(input_pdb_path),
+        f"sequence_info_{input_scfv_chain_id}_"
+        + basename(input_pdb_path)
+        + safety
+        + ".txt",
+    )
+
+    if not isfile(scfv_sequence_filename):
+        with open(scfv_sequence_filename, "wt") as f:
+            f.write(f">scfv_{input_scfv_chain_id}:...\n{scfv_seq}\n")
+
+    # run anarci:
+    anarci_executable = join(dirname(sys.executable), "ANARCI")
+    if not isfile(anarci_executable):
+        raise Exception(
+            f"ANARCI binary not found in {dirname(sys.executable)}. check installation"
```

Comment:
is it part of our requirements? if not add here an explanation of how to install.